src/github/github.js

```diff
@@ -1,7 +1,8 @@
 import "chrome-extension-async";
 import gitHubInjection from "github-injection";
 
-import { getOrbitCredentials, isRepoInOrbitWorkspace } from "./orbit-helpers";
+import { getOrbitCredentials } from "../oauth_helpers";
```

Comment:
Nitpick: can we use kebab case for this file too (`oauth-helpers.js`)? It looks like we’re using that convention throughout.